tools/static-assets/server/server-json.js

```diff
@@ -1,4 +1,6 @@
 var fs = require("fs");
 var path = require("path");
 var serverJsonPath = path.resolve(process.argv[2]);
-module.exports = JSON.parse(fs.readFileSync(serverJsonPath, 'utf8'));
+module.exports = JSON.parse(
+  fs.readFileSync(serverJsonPath, 'utf8').normalize('NFC')
```

Comment:
I guess this isn't using the new `unicodeNormalizePath` because it's not a path?